### DISCUSSION TITLE
Creating a LibRetro Frontend in Rust - Retro Reversing (Reverse Engineering)

### DISCUSSION THREAD

**Discussion by giscus** [RetroReversing/retroReversing#134] [Post Comments]
# Creating a LibRetro Frontend in Rust - Retro Reversing (Reverse Engineering)

RetroReversing, the home of retro game reverse engineering. Get resources, tutorials, and tools for analyzing video games and join the community today!

https://www.retroreversing.com/CreateALibRetroFrontEndInRust

**Comment by rishooty** [comment]
I'm not sure what happened to my prior comment, but my updates are finished!
https://github.com/rishooty/rustretro-tutorial

Any further changes I want to make are going to be made in a clone, as they go beyond the scope of this tutorial. For example, I'm replacing minifb with miniquad and rodio with quad-snd.

**Reply by RetroGameDeveloper** [reply]
This is great thank you! Do you mind if I use some of your changes as the basis for updates to this tutorial?

**Reply by rishooty** [reply]
> This is great thank you! Do you mind if I use some of your changes as the basis for updates to this tutorial?

Not at all! Please do.

**Comment by LukasBombach** [comment]
Hey, just here to say thank you! This is so awesome!
